### PR TITLE
Waltz 1584

### PR DESCRIPTION
**Pull request by davidwatkins73**
#1584

### CODE REVIEW

**Comment by davidwatkins73** [comment]
image

**Comment by davidwatkins73** [comment]
@rovats pls review

File: waltz-ng/client/survey/survey-template-list.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                        <a ui-sref='main.survey.template.view ({id: template.id})'
+                           class="clickable"
+                           ng-bind="template.name">
+                        </a>
+                    </td>
+                    <td>
+                        <span ng-bind="template.status"></span>
+                    </td>
+                    <td>
+                        <span ng-bind="template.createdAt"></span>
```
Comment (rovats) [inline waltz-ng/client/survey/survey-template-list.html]: `waltz-from-now`?

Comment (davidwatkins73) [inline waltz-ng/client/survey/survey-template-list.html]: done

**Pull request merged**